src/workerd/jsg/jsg.h

```diff
@@ -201,6 +201,18 @@ namespace workerd::jsg {
     registry.template registerMethod<NAME, decltype(&Self::name), &Self::name>();                  \
   } while (false)
 
+// Similar to JSG_METHOD, but also creates a V8 Fast API path for the method if its signature
+// is compatible with V8's Fast API Call conventions. This can significantly improve performance
+// for performance-critical methods with supported argument types.
+//
+// The same method will be used for both fast and slow paths, so it should be compatible with both.
```

Comment:
Could we just automatically use fast API for method signatures that support it, without the need for each method to be explicitly declared to use it?

Reply:
Yes, I'll add this to the TODO.

Reply:
I think ideally I'd prefer this to be addressed now as opposed to introducing the second macro here only to remove it later once the `JSG_METHOD` macro is updated.

Reply:
Yes, I'll address this before merging. I've added it to the TODO list in the pull-request description